text/0000-document_all_features.md

```diff
@@ -0,0 +1,289 @@
+
+3. This may delay landing features on stable. However, all the points raised in [**Precedent**][precedent] on this apply, especially:
+
+    > We can't get the great new toys unless everybody can enjoy the toys. ([@eccegordo])
+
+    For Rust to attain its goal of *stability without stagnation*, its documentation must also be stable and not stagnant.
+
+4. If the forthcoming docs team is unable to provide significant support for the core team member responsible for documentation, and perhaps equally if the rest of the community does not also increase involvement, this will simply not work. No individual can manage all of these docs alone.
+
+5. Specific to the suggestion to [**Add an "edit" link**][edit-link]:
+
+    - If the specific page is in flux (e.g. being rewritten, broken into pieces, etc.), then a link to edit `master` will be confusing.
+    - In addition, when users *have* made edits, it may take some time before it appears, and thus users may be confused when attempting to make edits and finding that the relevant editss have already been made.
+    - Some pages users attempt to edit are *likely* to have different documentation in them than the existing pages, to account for inbound changes for feature additions to the language!
+
+    Two notes, however:
+
+    1. Even facing the same issues, the Ember team has found it useful to have the link, as it enables basically any user of a sufficient comfort level with GitHub to fix basic typos or logic errors.
+    2. This concern primarily impacts _The Rust Programming Language_. Both in its current state and in the event of an eventual revamp (at least: after such a revamp finished), the Rust Reference is far less likely to see pages removed or moved.
+
+    Finally, while infrastructure changes could be made in support of a more "targeted" editing experience, doing so would substantially increase the triage work required for the docs. It would also entail extra work "porting" the changes back to `master`. Additionally, because the language itself does not currently "bugfix" releases, this would substantially alter the workflow for dealing with releases in general.
+
+6. Specific to the suggestion to [**Visually Distinguish Nightly**][distinguish-nightly]:
+
+    This requires at least some infrastructure investment. Making the change apply to the Reference as well as to the two books would entail the maintenance of further CSS. This might be acceptable if documentation teams are sufficiently motivated and engaged, but it means that if not very carefully designed up front, any changes to the documentation theme will basically require double CSS changes; they will also require double the *design* efforts.
```

Comment:
Perhaps this could be handled by just _adding_ a style-sheet. Stylesheets combine just fine and additional stylesheet defining a watermark and perhaps a couple of icons with `:before` and `:after` would be plenty conspicuous with just a few rules and shouldn't need too much maintenance.